### ISSUE TITLE
Using Goosh with Stripe module, GSON version?

### ISSUE THREAD

**Issue by max1020** [caffeinalab/ti.goosh#79]
I would like to use this module with Ravindra's Stripe (https://github.com/RavindraChherke/Stripe-module-for-Appcelerator-Titanium-iOS-and-Android). Both modules work on their own, but when I use both of them i get this notorious dexer issue where there is a conflict of libraries being used by the two modules:

https://medium.com/all-titanium/deal-with-titanium-modules-and-its-missing-support-for-android-multidex-546de5486d13
https://github.com/caffeinalab/ti.goosh/tree/master/gps-jars
https://stackoverflow.com/questions/39226890/multidex-error-when-using-ti-map-and-ti-goosh-module 

In this case the Stripe module uses gson 2.2.4 version. This module does not have the version in the name of the gson file. what version is it? Ive looked into the timodule file, but couldnt find the version. 

Any suggestions on how to proceed from here?

**Comment by m1ga** [comment]
Did you try to remove one of the gson.jar's ?

**Comment by m1ga** [comment]
I also updated gson to the latest version: https://github.com/caffeinalab/ti.goosh/pull/83

**Comment by m1ga** [comment]
Update for the stripe module: https://github.com/RavindraChherke/Stripe-module-for-Appcelerator-Titanium-iOS-and-Android/pull/9

**Comment by max1020** [comment]
Great! I will post some feedback on this later. many thanks!

**Comment by max1020** [comment]
I finally managed to test this (working on another project now).

Using version 3.0.1 of this module and the Stripe module (Titanium SDK >= 6) https://github.com/RavindraChherke/Stripe-module-for-Appcelerator-Titanium-iOS-and-Android/releases work together (when I remove the jar in this project)

I did not actually try sending push notifications but using those 2 modules (without the gson.jar of goosh) I am able to get the push notification device token, unregister it and get a new token.  Getting the stripe token works too.

Many thanks!
